cli/flox/src/commands/mod.rs

```diff
@@ -910,6 +911,9 @@ enum InternalCommands {
     /// Publish packages
     #[bpaf(command, hide, footer("Run 'man flox-publish' for more details."))]
     Publish(#[bpaf(external(publish::publish))] publish::Publish),
+    /// Upload packages
+    #[bpaf(command, hide, footer("Run 'man flox-upload' for more details."))]
```

Comment:
nit: `man flox-upload` wont actually work yet but i guess that'll come eventually if we keep and publicate this command...